packages/hash/integration/src/tests/model/knowledge/user.model.test.ts

```diff
@@ -36,7 +37,7 @@ describe("User model class", () => {
 
   let createdUser: UserModel;
 
-  let kratosIdentityId: string;
+  let kratosIdentityId: EntityId;
```

Comment:
This isn't an EntityId as it comes from kratos

Reply:
Pushed a fix